tensilelite/Tensile/Source/client/include/TypedId.hpp

```diff
@@ -115,5 +118,9 @@ namespace Tensile
     using TypedGemm_B8F8_S_S  = TypedGemm<BFloat8, Float8, float, float>;
     using TypedGemm_F8B8_B8_S = TypedGemm<Float8, BFloat8, BFloat8, BFloat8, float, float>;
     using TypedGemm_B8F8_B8_S = TypedGemm<BFloat8, Float8, BFloat8, BFloat8, float, float>;
+#ifdef TENSILE_USE_HALF
+    using TypedGemm_HF8_F8_S = TypedGemm<Half, Float8, Float8, Float8, float, float>;
```

Comment:
better to add this config with F8 impl PR

Reply:
Fixed